packages/test/test-version-utils/src/compatUtils.ts

```diff
@@ -93,9 +99,16 @@ function createGetDataStoreFactoryFunction(api: ReturnType<typeof getDataRuntime
 	};
 }
 
-// Only support current version, not baseVersion support
+/**
+ * Only support current version, not baseVersion support.
```

Comment:
This comment is strange too. I'm not sure how it relates to the variable, but it isn't a semantic description. I think we should move this back out as a `//` comment too.